### DISCUSSION TITLE
Wazuh distributed installation

### DISCUSSION THREAD

**Discussion by itdeveloper50** [wazuh/wazuh#15494] [Q&A] [answered]
I have one main site and 10 branches.

Is it possible to install wazuh like this:
- 1 management server (main server) on main site;
- 10 sensor nodes (forward nodes) at branches;
- create users on main site and give them access to be able to see only their sensor nodes' alerts (RBAC).

Does wazuh support above architecture?

Thank you!

**Comment by mdiego92** [comment, marked as answer]
Hello @itdeveloper50,

You want to have 1 main server, 10 sensor nodes that would be agents sending information to the main server, and different users with specific permissions regarding the 10 sensor nodes' data. Is that correct?

You may be able to redirect data from the nodes to the main server with rsyslog.
Article for reference: https://wazuh.com/blog/how-to-configure-rsyslog-client-to-send-events-to-wazuh/

For the user permissions, you can check the following article: https://documentation.wazuh.com/current/user-manual/wazuh-dashboard/rbac.html#use-case-give-a-user-permissions-to-read-and-manage-a-group-of-agents

Please let me know if I understood properly your request or if you have any remaining questions.

Thank you very much!

**Reply by itdeveloper50** [reply]
Hello @mdiego92 ,
Very close approach.
Yes, I will have 1 main server and 10 sensor nodes.
The function of sensor nodes is to collect all logs at branch and then send it to main server.
Is it possible?

**Reply by mdiego92** [reply]
Yes, @itdeveloper50. As I stated previously, you will be able to redirect data via `rsyslog`, or if it is possible, you can install Wazuh Agent in each node and set up the `logcollector` to send the information to the Manager as well.

Logcollector article for reference: https://documentation.wazuh.com/current/user-manual/capabilities/log-data-collection/how-it-works.html#log-collection

Please let me know if you have any remaining questions. I will be happy to answer them.
Thanks!

**Reply by itdeveloper50** [reply]
Thank you, I will install and check it.
